src/database/schema/ChatSchema.ts

```diff
@@ -59,6 +62,7 @@ class ChatSchema implements BaseDbSchema {
     this.status = status;
     this.isMe = isMe;
     this.isContinuous = isContinuous;
+    this.attachmentJson = attachmentJson;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` property is assigned without any validation or transformation. It would be safer to ensure that `attachmentJson` is always stored as a stringified JSON object.
```diff
-    this.attachmentJson = attachmentJson;
+    this.attachmentJson = JSON.stringify(attachmentJson);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->